lib/rust/ensogl/core/src/debug/monitor.rs

```diff
@@ -307,25 +622,79 @@ impl Renderer {
         }
     }
 
+    fn on_label_click(&mut self, position: Vector2) {
+        let panel_index = ((position.y as f64
+            - PADDING_TOP
+            - self.user_config.outer_margin
+            - self.user_config.margin / 2.0)
+            / (self.user_config.margin + self.user_config.panel_height))
+            .floor() as usize;
+        let panel_index = panel_index.max(0).min(self.panels.len() - 1);
```

Comment:
`.max(0)` should be done *before* converting to an unsigned type

Reply:
Oh, good point!